.github/workflows/basic.yml

```diff
@@ -130,6 +140,8 @@ jobs:
           -DUMF_FORMAT_CODE_STYLE=OFF
           -DUMF_DEVELOPER_MODE=ON
           -DUMF_BUILD_LIBUMF_POOL_DISJOINT=ON
+          -DUSE_ASAN=${{matrix.sanitizers}}
+          -DUSE_UBSAN=${{matrix.sanitizers}}
```

Comment:
I've added the ThreadSanitizer. MemorySanitizer requires some prework to work correctly (details here: https://github.com/google/sanitizers/wiki/MemorySanitizerLibcxxHowTo). After consulting people on stand-up, we've reached a conclusion that it would be best to add MemorySanitizer as an option but not include it in the CI.

Reply:
ok but how about leak sanitizer?